influxdb/line_protocol.py

```diff
@@ -119,32 +118,29 @@ def make_lines(data, precision=None):
         key_values = [measurement]
 
         # add tags
-        if static_tags is None:
-            tags = point.get('tags', {})
+        if static_tags:
+            tags = dict(static_tags)  # make a copy, since we'll modify
+            tags.update(point.get('tags') or {})
         else:
-            tags = copy(static_tags)
-            tags.update(point.get('tags', {}))
+            tags = point.get('tags') or {}
 
         # tags should be sorted client-side to take load off server
-        for tag_key in sorted(tags.keys()):
+        for tag_key, tag_value in sorted(iteritems(tags)):
             key = _escape_tag(tag_key)
-            value = _escape_tag(tags[tag_key])
+            value = _escape_tag(tag_value)
 
             if key != '' and value != '':
-                key_values.append("{key}={value}".format(key=key, value=value))
+                key_values.append(key + "=" + value)
```

Comment:
`key_values.append("=".join([key, value]))` would be faster actually.

String concatenation creates temporary strings for each concatenation and will get slower as strings gets larger. 

`join` OTOH is implemented in C and does the whole operation in one go.